src/scorers.rs

```diff
@@ -35,6 +35,12 @@ impl Score {
         }
         self.0 = value;
     }
+
+    /// Set the `Score`'s value. Allows values outside the range `0.0..=1.0`
+    /// Note: it is up to you to make sure your scorers are using the same units!
```

Comment:
I'd like a bit more of a warning here about the implications: It can make scorers much, much harder to effectively compose together when there isn't a single set scale.